src/Plotter/model_performance.py

```diff
@@ -0,0 +1,105 @@
+import seaborn as sns 
+import matplotlib.pyplot as plt
+import pandas as pd 
+from abc import ABC, abstractmethod
+
+class PlotModelPerformance:
+    def __init__(self) -> None:
+        pass
+
+    def create_accuracy_bar_plot(self, accuracy_df, title):
+        # Plotting
+        plt.figure(figsize=(8, 5))
+        plt.bar(accuracy_df['Accuracy Level'], accuracy_df['Accuracy'], color=['blue', 'orange', 'green'])
+        plt.xlabel('Match Level')
+        plt.ylabel('Accuracy (%)')
+        plt.title(title)
+        plt.ylim(0, 100)
+        plt.show()
+        return 
+
+    @abstractmethod
```

Comment:
is it meaningful to use the @abstractmethod decorator if this class is not an abstract class?

Reply:
Previously I though we might use this as a base class for model performances of different models. But for now I will remove the abstract method.

Reply:
alright - looks like there is another one to remove